docs/content/v0.20/contributor.md

```diff
@@ -0,0 +1,61 @@
+# KubeStellar Contributors
+
+**NOTE**: work in progress... write more
+
+## Check pre-requisites for KubeStellar
+
+The [check_pre_req](../../../hack/check_pre_req.sh) script offers a convenient way to check for pre-requisites, including the ones needed for [KubeStellar](./pre-reqs.md) deployment and [use case scenarios](./examples.md).

```

Comment:
The existing pre-reqs.md file covers prerequisites for more than "deployment". Have a look at the categories there, and the way the reference to ArgoCD is qualified.